modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-google-cloud-vertex-ai.adoc

```diff
@@ -0,0 +1,152 @@
+
+[source, bash, role="content-editable"]
+----
+#! /bin/bash
+
+cd /home/jupyter
+mkdir teradata
+cd teradata
+gsutil cp gs://teradata_jupyter/* .
+unzip teradatasql*.zip
+
+# Install Teradata kernel
+cp teradatakernel /usr/local/bin
+
+jupyter kernelspec install ./teradatasql --prefix=/opt/conda
+
+# Install Teradata extensions
+pip install --find-links . teradata_preferences_prebuilt
+pip install --find-links . teradata_connection_manager_prebuilt
+pip install --find-links . teradata_sqlhighlighter_prebuilt
+pip install --find-links . teradata_resultset_renderer_prebuilt
+pip install --find-links . teradata_database_explorer_prebuilt
+
+# PIP install the Teradata Python library
+pip install teradataml==17.20.00.04
+
+# Install Teradata R library (optional, uncomment this line only if you use an environment that supports R)
+#Rscript -e "install.packages('tdplyr',repos=c('https://r-repo.teradata.com','https://cloud.r-project.org'))"
+
+# Clone the Teradata lake-demos repository
+su - jupyter -c "git clone https://github.com/Teradata/lake-demos.git"
+----
+* 5. Upload this script to your Google Cloud storage bucket as a file
+
+image::{dir}/upload.png[files uploaded to bucket,align="center" width=75%]
+
+=== Initiating a user managed notebook instance
+
+* 1.Access Vertex AI Workbench
+
+- Return to Vertex AI Workbench in Google Cloud console.
+- Create a new User-Managed Notebook via Advanced Options or directly at https://notebook.new/.
+
+* 2. Under Details, name your notebook, select your region and select continue.
+
+image::{dir}/detailsenv.png[notebook env details,align="center" width=75%]
+
+* 3. Under Environment select **Browse** to select your startup.sh script from your Google Cloud Bucket.
+
+image::{dir}/startupscript.png[select startup script,align="center" width=75%]
+
+* 4. Select Create to initiate the notebook. It may take a few minutes for the notebook creation process to complete. When it is done, click on OPEN JUPYTERLAB. 
+
+image::{dir}/activenotebook.png[active notebook,align="center" width=75%]
+
+* On JupyterLab open a notebook with a Python kernel and run the following command for finding your notebook instance IP address.
+
+image::{dir}/python3.png[python3 kernel,align="center" width=75%]
+
+** We will whitelist this IP in your VantageCloud Lake environment in order to allow the connection.
+** This is for purposes of this guide and the notebooks demos. For production environments, a configuration of VPCs, Subnets and Security Groups might need to be configured and whitelisted.
+
+[source, python, role="content-editable"]
+----
+import requests
+def get_public_ip():
+    try:
+        response = requests.get('https://api.ipify.org')
+        return response.text
+    except requests.RequestException as e:
+        return "Error: " + str(e)
+my_public_ip = get_public_ip()
+print("My Public IP is:", my_public_ip)
+----
```

Comment:
This section feels a bit out of order. We say to run a snippet, then offer 2 more bullet points and then the snippet. Perhaps the bullet points should become a callout?